tests/jballerina-unit-test/src/test/resources/test-src/expressions/rawtemplate/raw_template_literal_test.bal

```diff
@@ -209,15 +209,15 @@ function testUseWithVar() {
     var rt = `Hello ${name}!`;
     typedesc<any> td = typeof rt;
 
-    assert("typedesc $rawTemplate$RawTemplate$_12", td.toString());
+    assert("typedesc $rawTemplate$RawTemplate$_14", td.toString());
```

Comment:
There is a query usage in this test file and order the lambdas are generated are changed now, So the number appended at the end of anonymous types is changed. I think this is some internal detail exposed outside. @HindujaB Hope this is ok?

Reply:
Eventhough, the `$_12` part is an internal one, the spec says about the `rawTemplate` object and its usage. 
Therefore, IMO it's okay to keep it like this for now.

We can later revisit this,  with a similar approach we use for anonymous records - providing the type descriptor instead of type name.